### ISSUE TITLE
Some WCAG2ICT content after word substitution does not match WCAG

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#412]
This content in WCAG2ICT was created by copy-and-paste from WCAG:

* success criteria with word substitutions
* glossary terms with word substitutions

In some cases, copy-and-paste based content was created before changes were made to WCAG. Examples of changes:
* [WCAG 2.1 errata corrections](https://www.w3.org/TR/WCAG21/#changelog)
* Any other changes not listed in the errata corrections (e.g., the change to 'changes of context')

In these cases, the errata corrections are appearing in WCAG2ICT in content pulled automatically from WCAG, but the errata corrections are not appearing in WCAG2ICT in the older copy-and-paste based content.

I pulled out all WCAG2ICT content with word substitutions, and diffed them with WCAG. The following differences should be fixed in WCAG2ICT.

- [x] 1.4.2 Audio Control: In WCAG the first instance of 'mechanism' is a link. In WCAG2ICT the second instance of 'mechanism' is a link.
- [x] 1.4.10 Reflow: Note 2 differs
- [x] 2.2.2 Pause, Stop, Hide: "Content" in Notes 2 and 3 has glossary links added in WCAG2ICT, which are not there in WCAG.
Above covered in PR https://github.com/w3c/wcag2ict/pull/468 
- [x] 2.4.3 Focus Order: 'navigated sequentially' should be a glossary link
- [x] 2.5.7 Dragging Movements: WCAG has four links to definitions that are not in WCAG2ICT
- [x] 2.5.8 Target Size (Minimum):
  - WCAG has "target" where WCAG2ICT has "target and target offset"
  - The last list item should end with a period, not a semicolon
  - In Note 1, WCAG has "sliders" where WCAG2ICT has "sliders with granular values"
 Above covered in PR https://github.com/w3c/wcag2ict/pull/469
- [x] 3.3.4 Error Prevention (Legal, Financial, Data): WCAG2ICT has added glossary links, which are not there in WCAG.
Above covered in PR https://github.com/w3c/wcag2ict/pull/470
- [x] Definition of 'accessibility supported': In bullet (1), 'content' has a glossary link added in WCAG2ICT, which is not there in WCAG
- [x] Definition of 'changes of context':
  - WCAG changed "major changes in the content of the Web page" to "major changes"
  - In bullet (4), 'content' has a glossary link in WCAG, which is not there in WCAG2ICT
- [x] Definition of 'relative luminance':
  - In WCAG2ICT, in Note 1, the text before the first unordered list has extra slash and asterisk characters.
  - WCAG changed the red threshold from 0.03928 to 0.04045
  - At the end of Note 1, ")." should be ".)"
  - WCAG has changed Note 6.
- [x] Definition of 'programmatically determined': "Software" has a glossary link added in WCAG2ICT, which is not there in WCAG.
- [x] Definition of 'same functionality': Two instances of the word 'Web' should be capitalized
- [x] Definition of 'user interface component': "content" has a glossary link added in WCAG2ICT, which is not there in WCAG.
- [x] Definition of 'viewport':
  - "content" has a glossary link added in WCAG2ICT, which is not there in WCAG.
  - Note 2 in WCAG2ICT is missing a footnote link to WCAG.
Above covered in PR https://github.com/w3c/wcag2ict/pull/471

- [x] A couple of more, not word substitutions, but direct quotes from WCAG: In 1.2.2 Captions (Prerecorded) and 1.2.4 Captions (Live), an added Note quotes the WCAG definition of 'captions'. The quotation has these issues:
    - WCAG has a link to the definition of 'text alternative' but WCAG2ICT does not. 
    - WCAG2ICT has a link to the definition of 'content' bug WCAG does not
    - Whitespace difference: "and/or" in WCAG, "and / or" in WCAG2ICT
    *Phil comment: I'm not sure I understand these quotation issues - note in applying SC 1.2.2. does not contain the phrase \'text alternative\' at all. * **Mary Jo comment: I found them and fixed in PR #477.

**Comment by mitchellevan** [comment]
#411 is relevant

**Comment by mitchellevan** [comment]
At first I assigned this issue to myself, but I'm out of time now to make the fix. It's up for grabs.

**Comment by mitchellevan** [comment]
See also #435

**Comment by ChrisLoiselle** [comment]
@maryjom I took this issue and made the tasks one proposal in the Google Doc. I wanted to have TF buy in before editing these.

**Comment by pday1** [comment]
@mitchellevan I've fixed all the issues noted above apart from the last group

> A couple of more, not word substitutions, but direct quotes from WCAG: In 1.2.2 Captions (Prerecorded) and 1.2.4 Captions (Live), an added Note quotes the WCAG definition of 'captions'. The quotation has these issues:
> 
> WCAG has a link to the definition of 'text alternative' but WCAG2ICT does not.
> WCAG2ICT has a link to the definition of 'content' bug WCAG does not
> Whitespace difference: "and/or" in WCAG, "and / or" in WCAG2ICT

I can't find these problems - so must have misinterpreted what the problems are. Are these issues in the applying SC (i.e. in the comments by guideline and success criterion), or in definitions, or somewhere else?

Thanks

**Comment by maryjom** [comment]
@pday1 I found those last problems and created a PR for that if you would please review.

**Comment by maryjom** [comment]
Thanks @mitchellevan for catching all of these detailed differences. The changes have been made and merged into the editor's draft, so closing this issue.
